README.md

```diff
@@ -481,29 +481,11 @@ Install to the workbench's virtualenv by running the following command form the
 pip install -r requirements.txt
 ```
 
-In the main XBlock repository, create the following configuration file
-in `workbench/settings_mentoring.py` in the XBlock repository:
-
-```python
-from settings import *
-
-INSTALLED_APPS += ('mentoring',)
-DATABASES['default']['NAME'] = 'workbench.sqlite'
-```
-
-Because this XBlock uses a Django model, you need to sync the database
-before starting the workbench. Run this from the XBlock repository
-root:
-
-```bash
-$ ./manage.py syncdb --settings=workbench.settings_mentoring
```

Comment:
@bradenmacdonald It would be nice to move to pure fields for answers, but there are then a few things to check:

* How do you maintain backward-compatibility with existing student data? Courses with existing answers will want to retain these answers. (This is btw a more general concern - existing data need to be retained for all fields that made use of persistence, ie things like assessment mode answers, max number of tries, etc.).
* Can data be accessed on XBlocks across different students? An important feature for Harvard is the ability to browse student answers from an admin dashboard - the initial naive implementation (put everything in a CSV) has been deactivated for performance reasons, but Harvard still wants it, and it will be back in the next set of changes as a dashboard allowing to browse the students answers. Would that be possible with pure XBlock fields?